### DISCUSSION TITLE
【第四期飞桨黑客松】产业合作开源贡献——基于PaddleDetection的无人机航拍图像检测

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50742] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#paddleindustry)
关于基于PaddleDetection的无人机航拍图像检测任务相关的讨论～

**Comment by yao-yu1999** [comment]
> [PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#paddleindustry) 关于基于PaddleDetection的无人机航拍图像检测任务相关的讨论～

这个赛道需要交提案吗？
